staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -667,6 +668,11 @@ func (s preparedGenericAPIServer) NonBlockingRun(stopCh <-chan struct{}, shutdow
 
 // installAPIResources is a private method for installing the REST storage backing each api groupversionresource
 func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *APIGroupInfo, openAPIModels *spec.Swagger) error {
+	typeConverter, err := fieldmanager.NewTypeConverter(openAPIModels, false)
```

Comment:
We used to have an instance per groupversion.  I'm not familiar enough with the fieldmanager to know if that matters offhand.

cc @apelisse

Reply:
We don't need multiple copies of these, they have the same input parameters and so are all going to be exact copies anyway. They are expensive to build and keep a lot of allocated memory in-use. They are considered immutable (except for some internal caching). And they all have the schema for all the internal types anyway. We're making their construction A LOT cheaper in #114439, but it's still much better to just do it once.